### DISCUSSION TITLE
NocoDB without prefix [Jun 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5927] [Discord Threads]
> 🙋‍♂️ **thomastraum**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### the prefix is added if it is a xcdb project. are you saying the prefix is also added to your ext db?   
####    
#### also ideally you should hv two seperate dbs - one for meta, and one for data.  
#  
> 🙋‍♂️ **thomastraum**  
#### thank you <@495466997304262658> I'll try again.  
#  
> 🙋‍♂️ **thomastraum**  
#### yes! I figured it out. the path is `/usr/app/data/yoursqlite.db`  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1106574186354315264)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
